File: WS/Models/IN/ContenusSearchParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WS.Models.IN
{
    public class ContenusSearchParameters
    {

        public Int32? Id { set; get; }
        public String Titre { set; get; }


        public String DtMin { set; get; }
        public String DtMax { set; get; }


        public Int32? EmplacementId { set; get; }
        public Int32? ModeId { set; get; }

        public Int32? EvenementId { set; get; }
        public Int32? TypologieId { set; get; }

    }

}

File: Angular/app/Contenus/Models/ContenusSearchParameters.ts
export class ContenusSearchParameters {

    constructor() {
        this.Id = null;

        this.Titre = null;

        this.DtMin = null;
        this.DtMax = null;

        this.EmplacementId = null;
        this.ModeId = null;

        this.EvenementId = null;
        this.TypologieId = null;

    }

    public Id: number | null;

    public Titre: string | null;

    public DtMin: string | null;
    public DtMax: string | null;

    public EmplacementId: number | null;
    public ModeId: number | null;

    public EvenementId: number | null;
    public TypologieId: number | null;


}